apps/bam.iobio/README.md

```diff
@@ -1,10 +1,40 @@
-# Overture Wrapper for Bam.Iobio Components
+# Overture Demo App for Bam.Iobio Components
 
-## Setup
+## Setup + Local Development
 
 - Install dependencies: `pnpm ci`
 
+- Start script: `pnpm run dev`
 
-## Local Development
+- App is running at `localhost:5173`
```

Comment:
will this port always be the same? is this a default set anywhere? what happens if the port is already being used?

Reply:
Very good questions, this is something internal to Vite, let me see

Reply:
Added env Ports [Add Port Env and update TS](https://github.com/overture-stack/iobio-components/pull/22/commits/5778d71db7c45695ab3003acfcc889732804557f)